### DISCUSSION TITLE
sign Bep20 token in android

### DISCUSSION THREAD

**Discussion by chamid** [trustwallet/wallet-core#2166] [Q&A]
Hi

I want to sign a Bep20 token but I couldn't find in android's tests and samples anything

Does core-wallet android sdk support this???

If yes can you please provide an example or related link???

Thanks

**Comment by UNTUNGSUTARNO** [comment]
Yes

UNTUNG SUTARNO

Pada tanggal Sel, 26 Apr 2022 19:42, chamid ***@***.***>
menulis:

**Comment by UNTUNGSUTARNO** [comment]
related link

UNTUNG SUTARNO

Pada tanggal Sel, 26 Apr 2022 20:03, UNTUNG SUTARNO ***@***.***>
menulis:

**Comment by optout21** [comment]
See sample test here:

https://github.com/trustwallet/wallet-core/blob/master/tests/BinanceSmartChain/SignerTests.cpp#L48
